backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/DockerRabbitMQ.java

```diff
@@ -79,8 +81,8 @@ private DockerRabbitMQ(Optional<String> hostName, Optional<String> erlangCookie,
 
     private WaitAllStrategy waitStrategy() {
         return new WaitAllStrategy()
-            .withStrategy(Wait.forHttp("").forPort(DEFAULT_RABBITMQ_ADMIN_PORT).withRateLimiter(RateLimiters.DEFAULT))
-            .withStrategy(RabbitMQWaitStrategy.withDefaultTimeout(this));
+            .withStrategy(Wait.forHttp("").forPort(DEFAULT_RABBITMQ_ADMIN_PORT).withRateLimiter(RateLimiters.TWENTIES_PER_MINUTE))
```

Comment:
Can we have a rate a bit bigger than 'one time every 3 sendonds' ? Ten check per second should be OK, no?

Reply:
What is the default value?

Reply:
Default timeout for http check is one minute. perform http request in high rate might make containers busy to handle http requests, and lead to not really up in one minute. one request per second is enough, Don't you think?

Reply:
Is what you report an observation or a speculation?

If it is an observation then yes let's wait that extra second to be stable.